layouts/partials/header.html

```diff
@@ -32,7 +32,7 @@
     {{ range .Site.Authors }}
       {{ with .twitter }}<meta name="twitter:creator" content="@{{ . }}"/>{{ end }}
     {{ end }}{{ end }}
-
+    <script async defer src="/js/lazysizes.min.js"></script>
```

Comment:
Note `defer` here is not necessary, but has been added as a convenience for older versions of Internet Explorer, which do not make use of the `async` attribute.